ls1intum/Artemis
# Bugfix: delete unused file

### Checklist

### Motivation and Context


The file was moved to another directory and probably duplicated by accident when resolving a merge conflict.

### Description

I deleted the file.

### Steps for Testing


1. Open console
2. Check that there isn't a `unused file` warning
